src/util/setSelectionsAndFocusEditor.ts

```diff
@@ -65,6 +68,26 @@ export async function focusEditor(editor: TextEditor) {
   }
 }
 
+function getViewColumn(editor: TextEditor): ViewColumn | undefined {
+  if (editor.viewColumn != null) {
+    return editor.viewColumn;
+  }
+  // TODO: tabGroups is not available on older versions of vscode we still support.
+  // Remove any cast as soon as version is updated.
+  if (semver.lt(version, "1.67.0")) {
+    return undefined;
+  }
+  const viewColumn: ViewColumn | undefined = (window as any)?.tabGroups
+    ?.activeTabGroup?.viewColumn;
```

Comment:
Why are you using the view column of the active tab group? Shouldn't you look for the tab group containing `editor`?

Reply:
As far as I can tell that information is not available on each tab. I have to dig deeper in the api and see if I can find that data.

Reply:
There is no editor or document on the tab groups, but I managed to find a uri